src/useKeyboardShortcuts.ts

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2022-2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { useCallback, useState } from "react";
+
+import { getSetting } from "./settings/useSetting";
+import { useEventTarget } from "./useEvents";
+
+export function useKeyboardShortcuts(
+  enabled: boolean,
+  toggleMicrophoneMuted: () => void,
+  toggleLocalVideoMuted: () => void,
+  setMicrophoneMuted: (muted: boolean) => void
```

Comment:
Tbh, it feels weird to have all of these in one hook, perhaps renaming the hook to `useCallViewKeyboardShortcuts` might make this better

Reply:
True - renamed